src/components/applications/ApplicationFilter.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { Filter, TextInput } from "react-admin";
+
+const ApplicationFilter = (props) => (
+  <Filter {...props}>
+    <TextInput label="Search by Name" source="applicationName" alwaysOn />
+    <TextInput label="Search by Organization" source="organizationName" />
```

Comment:
Since these are filters, I would suggest that you label them as such. Given that the labels aren't strictly titles, I would also suggest that you only make the first word's first letter uppercase.

```suggestion
    <TextInput label="Filter by name" source="applicationName" alwaysOn />
    <TextInput label="Filter by organization" source="organizationName" />
```